modules/persistent-storage-snapshots-starting-controller-provisioner.adoc

```diff
@@ -46,6 +46,12 @@ rules:
     resources: ["volumesnapshotdatas"]
     verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
 ----
+. As the cluster admin, provide the `hostNetwork security context` constraint (SCC):
+----
+# oc adm policy add-scc-to-user hostnetwork -z snapshot-controller-runner
+----
+This SCC controls access to the `snapshot-controller-runner` ServiceAccount that the Pod is using.
```

Comment:
* This needs a + above it as well to stay indented. Note that missing these +'s makes the next step restart numbering at 1.
* ServiceAccount -> service account [1]

[1] https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/term_glossary.adoc#service-account

Reply:
Finally got the +'s in the right places, thanks!